components/hab/src/command/studio/docker.rs

```diff
@@ -64,12 +64,24 @@ pub fn start_docker_studio(_ui: &mut UI, args: &[OsString]) -> Result<()> {
     } else {
         ""
     };
+
+    let local_cache_key_path = default_cache_key_path(None);
+    if !local_cache_key_path.exists() {
+        return Err(Error::FileNotFound(
+            format!(
+                "{}\nRun `hab setup` to create an origin or \
+                use `hab origin key` to configure your keys.",
+                local_cache_key_path.to_string_lossy().into_owned()
```

Comment:
Both here and on L84, I think [`PathBuf::display`](http://doc.rust-lang.org/1.34.1/std/path/struct.PathBuf.html#method.display) would be preferable to `to_string_lossy`. Also, think we can omit `into_owned`.